.github/aboutme.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!-- title -->
+<head>
+  <meta charset="UTF-8">
+  <title>Muzart's Awesome Kitchen</title>
+  <link rel="stylesheet" href="website.css">
+</head>
+  <!-- rest of page -->
+<body>
+  <header class="header">
+    <a class="title-container" href="index.html">
+      <h1 class="title">Muzart's Awesome Kitchen</h1>
+    </a>
+    <nav>
+      <a class="nav-item" href="index.html">Home</a>
+      <a class="nav-item" href="aboutme.html">About The Chef</a>
+    </nav>
+  </header>
+  <main>
+    <h1>About The Chef</h1>
+    <p>I am okay at cooking</p>
+    <img class="img-large" src="/Users/muzarttuman/Downloads/gettyimages-1081422898-612x612.jpeg" alt="chef">
```

Comment:
Use relative paths for images, otherwise it only works on your computer. This means that you must also commit your images to GitHub